text/0061-relax-mixin-contract.md

```diff
@@ -3,6 +3,7 @@
 - Name: Relax Mixin Contract
 - Start Date: 2020-08-12
 - Author(s): @sclevine
+- Status: Approved
```

Comment:
```suggestion
- Status: Superseded
```
Is this superseded by RFC 0096?

Reply:
I apologize for missing this.

_Addressed in latest commit._